CMakeLists.txt

```diff
@@ -96,10 +96,6 @@ option(IREE_ENABLE_NEW_INTEGRATION_TESTS "Enables new integration tests and disa
 # Derived flags based on primary options
 #-------------------------------------------------------------------------------
 
-if(${IREE_BUILD_SAMPLES})
-  set(IREE_BUILD_COMPILER ON CACHE BOOL "Build the IREE compiler for sample projects." FORCE)
-endif()
```

Comment:
Hmm... https://source.cloud.google.com/results/invocations/2a89b97a-30df-4923-a8fd-e922e98b0936/targets/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Friscv64%2Fpresubmit/log

https://github.com/google/iree/blob/main/iree/samples/simple_embedding/CMakeLists.txt needs more surgery so https://github.com/google/iree/blob/main/build_tools/cmake/build_riscv.sh builds that sample.